schema/rdl_schema_0.1.json

```diff
@@ -90,14 +90,13 @@
 							"type": "string",
 							"description": "Additional details about the dataset"
 						},
-						"biblio_title": {
-							"type": "string",
-							"description": "Authors/year/title of publication"
-						},
-						"biblio_url": {
-							"type": "string",
-							"description": "URL of publication",
-							"format": "uri"
+						"bibliography": {
+							"title": "Bibliography",
+							"description": "An array of one of more publications that relate to or have made use of this dataset",
```

Comment:
I think that we need to clarify the semantics of this field. A bibliography is a list of the publications referenced in scholarly work. However, according to the the description, we're talking about the reverse of that, i.e. a list of publications that reference this dataset.

Reply:
Also we should avoid restating the type of a field in its description.

Reply:
Continued in https://github.com/GFDRR/rdl-standard/issues/58